imsv-docs-astro/astro.config.mjs

```diff
@@ -36,6 +36,7 @@ export default defineConfig({
           label: 'Guides',
           items: [
             { label: 'Introduction', link: 'guides' } ,
+            { label: 'Regions', link: 'guides/regions' } ,
```

Comment:
Please move next to "supported chains".

Reply:
done that, as well as renamin to 'Supported Regions'